### PR TITLE
SH-561 env variable for telemetry

### PR DESCRIPTION
**Pull request by sknirmalkar89**

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sunbird_groups-service.env [hunk 1]
```diff
@@ -27,5 +27,6 @@ accesstoken.publickey.keyprefix={{ adminutil__access_keyprefix | default('access
 accesstoken.publickey.keycount={{ adminutil__access_keycount | default(10)}}
 sunbird_user_service_search_url=/private/user/v1/search
 sunbird_cs_search_url=/v3/search
-
+ENV_NAME={{env_name | default('dev')}}
+SUNBIRD_KAFKA_URL={{sunbird_kafka_url | default('localhost:9092')}}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_groups-service.env:31]: We dont have any such varaible named **sunbird_kafka_url**
We dont want to introduce new variables. Please reuse the variable **kafka_urls**
You can refer **ansible/roles/stack-sunbird/templates/sunbird_learner-service.env** for this

File: ansible/roles/stack-sunbird/templates/sunbird_groups-service.env [hunk 2]
```diff
@@ -27,5 +27,6 @@ accesstoken.publickey.keyprefix={{ adminutil__access_keyprefix | default('access
 accesstoken.publickey.keycount={{ adminutil__access_keycount | default(10)}}
 sunbird_user_service_search_url=/private/user/v1/search
 sunbird_cs_search_url=/v3/search
-
+ENV_NAME={{env_name | default('dev')}}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_groups-service.env:30]: Please dont defined a default for **env_name**
You can refer **ansible/roles/stack-sunbird/templates/sunbird_learner-service.env** for this.

File: ansible/roles/stack-sunbird/templates/sunbird_groups-service.env (hunk 1 above)
Comment (sknirmalkar89) [reply ansible/roles/stack-sunbird/templates/sunbird_groups-service.env]: updated

File: ansible/roles/stack-sunbird/templates/sunbird_groups-service.env (hunk 2 above)
Comment (sknirmalkar89) [reply ansible/roles/stack-sunbird/templates/sunbird_groups-service.env]: updated

**Pull request merged**